src/velero/zarf.yaml

```diff
@@ -34,6 +35,7 @@ components:
       - registry1.dso.mil/ironbank/opensource/velero/velero:v1.14.0
       - registry1.dso.mil/ironbank/opensource/velero/velero-plugin-for-aws:v1.10.0
       - registry1.dso.mil/ironbank/big-bang/base:2.1.0
+      - registry1.dso.mil/ironbank/opensource/velero/velero-plugin-for-microsoft-azure:v1.10.0
 
   - name: velero
     required: true
```

Comment:
I think it's _okay_ to leave azure plugin off of the unicorn flavor for now but would like to at least track that with an issue. The alternative is for us to add the upstream image here but it has several medium/high/cricital CVEs 😒.

Reply:
Copy that. I'll create a separate issue that tracks the updates needed to the unicorn flavor (tied to the ticket opened with Chainguard to provide the image).

Reply:
Tracking [this issue](https://github.com/defenseunicorns/uds-core/issues/674) to resolve this. Also there is are pending PRs for the azure plugin that would bump the go version and azure sdk version which would resolve all CVEs. If that does get merged it may make sense to add the upstream image here to start.